core-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/SnapshotFileBatchSink.java

```diff
@@ -108,7 +108,7 @@ public void onRunFinish(boolean succeeded, BatchSinkContext context) {
           long cutoffTime =
             context.getLogicalStartTime() - TimeParser.parseDuration(config.getCleanPartitionsOlderThan());
           snapshotFileSet.deleteMatchingPartitionsByTime(cutoffTime);
-          LOG.info("Cleaning up snapshots older than {}", cutoffTime);
+          LOG.debug("Cleaning up snapshots older than {}", cutoffTime);
```

Comment:
This should be left as info since its based on the user config. Should be changed to better readable format than the long.